docs/deploy.md

```diff
@@ -38,23 +38,62 @@ Make sure that your Kubernetes config (e.g, `~/.kube/config`) is pointing to the
    of the Istio Ingress Gateway service. You can retrieve the external IP of this service by running
    `kubectl get svc -n istio-system istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[*].ip}'`
    1. If you used the `./hack/generate-values.sh` script then you should only
-      configure a single DNS record for the domain you passed as input to the
-      script and have it resolve to the Ingress Gateway's external IP
+      configure a single DNS record for the wildcard subdomain of the domain
+      you passed as input to the script and have it resolve to the Ingress
+      Gateway's external IP
 
 ## Validate the deployment
 
-1. Set up cf cli to point to CF:
+1. Set up cf CLI to point to CF:
    ```bash
    $ cf api --skip-ssl-validation https://api.<system_domain>
-   $ cf auth admin <cf_admin_password>
+   $ cf auth admin $(bosh interpolate /tmp/cf-values.yml --path /cf_admin_password)
```

Comment:
Let's keep the angle-bracketed line and suggest that the value for `cf_admin_password` can be obtained through the `bosh interpolate` expression.

This keeps with the convention used in the rest of the instructions.

Reply:
@paulczar ^^